gointelowl/client.go

```diff
@@ -37,11 +38,11 @@ type successResponse struct {
 }
 
 type IntelOwlClientOptions struct {
-	Url   string
-	Token string
+	Url   string `json:"url"`
+	Token string `json:"token"`
 	// * so basically your SSL cert: path to the cert file!
-	Certificate string
-	Timeout     time.Duration
+	Certificate string `json:"certificate"`
+	Timeout     uint64 `json:"timeout"`
```

Comment:
Is this timeout in seconds or milli-seconds?

Reply:
Seconds. Yeah I'll make a comment that it is in seconds.